test/mailers/previews/interview_mailer_preview.rb

```diff
@@ -0,0 +1,12 @@
+# Preview all emails at http://localhost:3000/rails/mailers/interview_mailer
+class InterviewMailerPreview < ActionMailer::Preview
+  def reminder_interviewee_mail
+    interview = Interview.find(2)
```

Comment:
?

Reply:
I just did this, just for testing, I should have to handle cases here. and use Interview.first, Sorry for that